sound/soc/sof/intel/byt.c

```diff
@@ -630,9 +630,9 @@ static int byt_acpi_probe(struct snd_sof_dev *sdev)
 	}
 
 	dev_dbg(sdev->dev, "using IRQ %d\n", sdev->ipc_irq);
-	ret = request_threaded_irq(sdev->ipc_irq, byt_irq_handler,
-				   byt_irq_thread, IRQF_SHARED, "AudioDSP",
-				   sdev);
+	ret = request_threaded_irq_devm(sdev->dev, sdev->ipc_irq,
```

Comment:
that looks wrong?

Reply:
@plbossart Sorry about that. I compiled the kernel but didn't select all platforms.